.github/workflows/NUnitConsoleAndEngine.CI.yml

```diff
@@ -1,118 +1,84 @@
-    #   with:
-    #     name: Package
-    #     path: package
-
-    # - name: 💾 Upload test results
-    #   uses: actions/upload-artifact@v4
-    #   with:
-    #     name: Test results (Windows)
-    #     path: test-results
-    #   # Use always() to always run this step to publish test results when there are test failures
-    #   if: ${{ always() }}
-
-  # build-linux:
-  #   name: Linux Build
-  #   runs-on: ubuntu-latest
-
-  #   steps:
-  #   - name: ⤵️ Checkout Source
-  #     uses: actions/checkout@v4
-
-  #   - name: 🛠️ Setup .NET
-  #     uses: actions/setup-dotnet@v4
-  #     with:
-  #       global-json-file: global.json
-
-  #   - name: 🛠️ Install F#
-  #     run: sudo apt-get install fsharp
-
-  #   - name: 🛠️ Install dotnet tools
-  #     run: dotnet tool restore
-
-  #   - name: 🔨 Build and Test
-  #     run: dotnet tool run dotnet-cake --target=Test --test-run-name=Linux --configuration=Release
-
-  #   - name: 💾 Upload test results
-  #     uses: actions/upload-artifact@v4
-  #     with:
-  #       name: Test results (Linux)
-  #       path: test-results
-  #     # Use always() to always run this step to publish test results when there are test failures
-  #     if: ${{ always() }}
-
-  # build-macos:
-  #   name: MacOS Build
-  #   runs-on: macos-14
-
-  #   steps:
-  #   - name: ⤵️ Checkout Source
-  #     uses: actions/checkout@v4
-
-  #   - name: 🛠️ Setup .NET
-  #     uses: actions/setup-dotnet@v4
-  #     with:
-  #       global-json-file: global.json
-  #       dotnet-version: 6.x
-
-  #   - name: 🛠️ Install dotnet tools
-  #     run: dotnet tool restore
-
-  #   - name: 🔨 Build and Test
-  #     run: dotnet tool run dotnet-cake --target=Test --test-run-name=Linux --configuration=Release
-
-  #   - name: 💾 Upload test results
-  #     uses: actions/upload-artifact@v4
-  #     with:
-  #       name: Test results (macOS)
-  #       path: test-results
-  #     # Use always() to always run this step to publish test results when there are test failures
-  #     if: ${{ always() }}
+      - name: ⤵️ Checkout Source
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: 🛠️ Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          # global-json-file: global.json
+          dotnet-version: |
+            2.1.x
+            3.1.x
+            6.0.x
+            8.0.100
+
+      - name: 🔧 Install dotnet tools
+        run: dotnet tool restore
+
+      - name: 🍰 Run cake
+        shell: pwsh
+        # If you need to get more verbose logging, add the following to the dotnet-cake above:  --verbosity=diagnostic
+        run: dotnet cake --target=Test --test-run-name=${{ matrix.testRunName }} --configuration=Release
+
+      - name: 🪵 Upload build logs
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: NUnitConsoleLogs
+          # This path is defined in build-settings.cake
+          path: "build/*.binlog"
```

Comment:
This file will not exist (and the step will error) until binlogs are enabled via nunit/NUnit.Cake.Recipe#5.